lib/sportradar/api/nfl/team.rb

```diff
@@ -36,6 +42,11 @@ def full_name
         [market, name].join(' ')
       end
 
+      def record
+        return '' unless wins && losses && ties
```

Comment:
Rather than hard return a blank string, just wrap your code in an if/unless statement so it will return `nil`; generally speaking, I don't like using hard returns unless we are trying to skip a big block of code from ever being executed. This is just stylistic